presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java

```diff
@@ -117,4 +123,14 @@ public static HiveStorageFormat getHiveStorageFormat(Map<String, Object> tablePr
     {
         return (List<String>) tableProperties.get(BUCKETED_BY_PROPERTY);
     }
+
+    public static String getFieldDelimProperty(Map<String, Object> tableProperties)
+    {
+        return StringEscapeUtils.unescapeJava((String) tableProperties.get(FIELD_DELIM_PROPERTY));
```

Comment:
Sometimes we create table with field delimiter '\t', the field delimiter of target table will be '\\t' without unescape. Is there a better way to handle this?